jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojo.java

```diff
@@ -248,7 +249,7 @@ private void generateSchema(Class<?> schemaClass) throws MojoExecutionException
         JsonNode jsonSchema = getGenerator().generateSchema(schemaClass);
         File file = getSchemaFile(schemaClass);
         this.getLog().info("- Writing schema to file: " + file);
-        this.writeToFile(jsonSchema, file);
+        this.writeToFile(getGenerator().getConfig().getObjectMapper(), jsonSchema, file);
```

Comment:
**suggestion**: I suppose we don't need the extra parameter here and can simply retrieve the `ObjectMapper` within the `writeToFile()` method directly. Both of these methods are `private`, i.e., don't have to cater for additional flexibility in regards to where the `ObjectMapper` is coming from.
```suggestion
        this.writeToFile(jsonSchema, file);
```